### ISSUE TITLE
PPA changed 'Label' value from 'EGPU Switcher' to 'egpu-switcher'

### ISSUE THREAD

**Issue by benjamin-rood** [hertg/egpu-switcher#39]
```
E: Repository 'http://ppa.launchpad.net/hertg/egpu-switcher/ubuntu bionic InRelease' changed its 'Label' value from 'EGPU Switcher' to 'egpu-switcher'
N: This must be accepted explicitly before updates for this repository can be applied. See apt-secure(8) manpage for details.
```
Tried removing the PPA and adding it again, doesn't fix it. I don't want to uninstall `egpu-switcher` altogether in order to fix this, because it's working so well for me! :hankey: 

p.s. Is there any place I can drop you some $ as a thank you for your work on this? My EGPU is more reliable than the internal card and HDMI port, that's how great this has worked for me.

**Comment by hertg** [comment]
Oops, I thought this was just a display name on launchpad.
I changed the label back to its original name "EGPU Switcher", does that fix the issue?

> p.s. Is there any place I can drop you some $ as a thank you for your work on this? My EGPU is more reliable than the internal card and HDMI port, that's how great this has worked for me.

I'm a bit overwhelmed, I have been asked that before, and I honestly don't know. I did look at the Github Sponsor program today, but this seems to be a monthly sponsor thing, and I don't feel like I'm doing enough to justify that. If you know of any suitable service, let me know.
Always feel free to donate the money to a charity you trust instead. :sweat_smile: 

Anyway, glad to hear that my script worked well for you :smile:
